### DISCUSSION TITLE
unstable_getServerSession vs getSession

### DISCUSSION THREAD

**Discussion by Jelle28** [nextauthjs/next-auth#5423] [Help]
In the configuration documentation of unstable_getServerSession ([here](https://next-auth.js.org/configuration/nextjs)) I've read the following:

> When calling from server-side i.e. in API routes or in getServerSideProps, we recommend using this function instead of getSession to retrieve the session object.

That's a clear statement, but the same page also states:

> WARNING This feature is experimental and may be removed or changed in the future.

I find those two statements conflicting. Is it recommend to use unstable_getServerSideProps instead of getSession for applications running as production?

**Comment by raulmarindev** [comment]
It would be great if we could get a clarification on this. We want to use it in our app but the warning makes us reticent.
